oak_runtime/src/lib.rs

```diff
@@ -463,7 +463,7 @@ impl Runtime {
             if node_info.abi_handles.get(&candidate).is_none() {
                 debug!(
                     "{:?}: new ABI handle {} maps to {:?}",
-                    self.get_node_debug_id(node_id),
+                    node_info.get_debug_id(node_id),
```

Comment:
Good catch, but also oh noes. 😬
I haven't worked with this multi threaded code in a while, but deadlocking the threads aren't really something I've started making peace with yet. It just seems so tricky to catch, it I don't like it